lib/utils/get-element-type.js

```diff
@@ -0,0 +1,36 @@
+const {elementType, getProp, getPropValue} = require('jsx-ast-utils')
+
+/*
+Allows custom component to be mapped to an element type.
+When a default is set, all instances of the component will be mapped to the default.
+If a prop determines the type, it can be specified with `props`.
+
+For now, we only support the mapping of one prop type to an element type, rather than combinations of props.
+*/
+function getElementType(context, node) {
+  const {settings} = context
+  const rawElement = elementType(node)
+  if (!settings) return rawElement
+
+  const componentMap = settings['github']?.components
```

Comment:
I think this is causing the CI failure - optional chaining is only supported in Node 13+.
```suggestion
  const componentMap = settings.github && settings.github.components
```

Reply:
Thank you!